lib/command.py

```diff
@@ -40,6 +41,19 @@ def say(self, player: Player, message: str):
                 # send them a message telling them what the player said
                 other_player.message(f"{player.name} says: {message}")
 
+    def tell(self, player: Player, params):
+        """ tell <player> message - Whisper a private message to the target player."""
+        target = params.split(' ')[0]
+        message = ' '.join(params.split(' ')[1:])
```

Comment:
This will throw an exception if the syntax is incorrect (ie we just get the command `tell`)

Reply:
It just returns 'There is nobody online by that name.' in that case.

Reply:
How come it's not an `IndexError` since there wouldn't be a second index on the split?

Reply:
At a first glance, I'm not entirely sure. :P